hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSWithObjectStoreCreate.java

```diff
@@ -302,10 +304,13 @@ public void testMPUFailDuetoDirectoryCreationBeforeComplete()
 
     // This should succeed, as we check during creation of part or during
     // complete MPU.
+    ozoneOutputStream.getMetadata().put("ETag",
+        DatatypeConverter.printHexBinary(MessageDigest.getInstance("Md5")
```

Comment:
We can unify the "Md5" using a constant variable.

Can be addressed in another ticket if needed.